test_accounts.md

```diff
@@ -2,165 +2,181 @@
 
 These are test accounts you can use to test OAuth APIs by logging in with them through the ID.me sandbox environment that is connected to sandbox-api.va.gov.  When signing in with a test account, Step 2 will ask how you want to receive an authentication code and you can just click "continue" for the verification to be automatically populated in Step 3.
 
+## Representative Oauth
+
+The purpose behind Representative Oauth is to allow you to login as one of 3 types of users (VSO, Attorney or Claim Agent) who holds power of attorney (PoA) for a particular veteran through the Oauth and ID.me system and then user the token you've collected to essentially access that Veteran's data (currently only available in our [Claims V1 API's](https://developer.va.gov/explore/benefits/docs/claims?version=current)), this process involves us looking at which Office of General Counsel (OGC) PoA code the veteran has and matching it to the PoA code the Representative is authorized to access. Example of a Representatie curl command is provided below.
```

Comment:
Typo in "Representative" at the end of this paragraph

Reply:
@ryanlink Is there 1 test account for PoA or 3? This copy makes me think there are 3 separate test accounts (one test user for VSO, one test user for Attorney and one test user for Claim Agent). But ticket API-670 has me thinking it's a single test account  (user 007) that is meant to encompass all representative OAuth users.

@mlvidart for test user awareness. Side convo to take offline - do we have user type/proto-persona for Claim Agent. This is the first time I'm hearing that user role in this context.

Reply:
We do! Here's the User-Types, we call them Claims Officers. They have different names in different organizations:
 https://github.com/department-of-veterans-affairs/vets-contrib/blob/master/products/developer-portal/research/user-types/descriptions-slides.md

Reply:
So I removed unecessary information from the table itself and updated the Representative Oauth section to reflect that here really is only one type of user here and that is a representative, whether they are a VSO, an attorney or a Claims Agent doesn't really matter, all that matters is whether or not they have PoA or not. Important to note that users 001 and 007 are both Representatives with access to the same test users, 001 is meant to represent an actual veteran that also for example is a VSO so they might have access to both their data through the system as well as that of those with PoA data etc while 007 is meant to only have representative access since they have no data of their own etc.